tools/walletextension/wallet_extension.go

```diff
@@ -348,6 +400,86 @@ func setCallFromFieldIfMissing(args []interface{}, account common.Address) ([]in
 	return args, nil
 }
 
+// Stores a viewing key to disk.
+func (we *WalletExtension) persistViewingKey(viewingKey *rpcclientlib.ViewingKey) {
+	viewingPrivateKeyBytes := crypto.FromECDSA(viewingKey.PrivateKey.ExportECDSA())
+
+	persistenceEntry := fmt.Sprintf("%s %s %s %s",
```

Comment:
Tab separated would be slightly safer/more conventional than space separated I think